realtime/server.js

```diff
@@ -153,7 +164,8 @@ socket.on('connection', function (client) {
 
     client.on('typing-update', function(data) {
         winston.debug('[' + people[client.id] + '] typing-update');
-
+        
+        //check for possible errors, while the message is on process
```

Comment:
The "while the message is on process" portion is obvious because we are within the event handler.